04/modules/API.js

```diff
@@ -0,0 +1,98 @@
+class API {
+    constructor() {
+        // to check real time data set "this.API.enabled" to "true"
+        this.API = {
+            url: 'https://api.weatherbit.io/v2.0/current?key=ffc13da65b494013b12370e7bdecb3f0',
+            enabled: false,
+        };
+        this.dataForAPI = {
+            lat: '',
+            lng: '',
+            units: 'I',
+            lang: 'pl',
+        };
+        this.apiProto = {
```

Comment:
Może dodałbym `...forTesting` lub `apiStub` - dlaczego taka nazwa to ten artykuł powinien Ci to rozjaśnić. Nie koniecznie musisz się wgryzać w sam kod ;) https://www.samouczekprogramisty.pl/testy-jednostkowe-z-uzyciem-mock-i-stub/

Reply:
Jeszcze nie wszystko z tego artykułu jest dla mnie zrozumiałe. Na szczęście niedługo przechodzimy do modułu o testowaniu, to na pewno mi się rozjaśni :). Dzięki!